commit 55dddf02a3ba8c5467a5f2d4803ead595a9cf446
Author: Ido Schimmel <idosch@mellanox.com>
Date:   Tue Feb 28 08:55:40 2017 +0100

    mlxsw: spectrum_router: Avoid potential packets loss
    
    [ Upstream commit f7df4923fa986247e93ec2cdff5ca168fff14dcf ]
    
    When the structure of the LPM tree changes (f.e., due to the addition of
    a new prefix), we unbind the old tree and then bind the new one. This
    may result in temporary packet loss.
    
    Instead, overwrite the old binding with the new one.
    
    Fixes: 6b75c4807db3 ("mlxsw: spectrum_router: Add virtual router management")
    Signed-off-by: Ido Schimmel <idosch@mellanox.com>
    Signed-off-by: Jiri Pirko <jiri@mellanox.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1383_55dddf02a3ba8c5467a5f2d4803ead595a9cf446.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1383_55dddf02a3ba8c5467a5f2d4803ead595a9cf446.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
